src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts

```diff
@@ -96,72 +96,78 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
       class:'general',
       width:'300px',
       config:[
-        { 
-          type: 'input', 
-          name: 'name', 
+        {
+          type: 'input',
+          name: 'name',
           width:'100%',
           placeholder: 'Custom Theme Name',
           required:true,
-          tooltip: 'Enter a name to identify your new theme.',
+          tooltip: 'Enter a name to identify the new theme.',
         },
-        { 
-          type: 'input', 
-          name: 'label', 
+        {
+          type: 'input',
+          name: 'label',
           width:'100%',
           placeholder: 'Menu Label',
           required:true,
-          tooltip: 'Specify how the theme name should appear in the menu.',
+          tooltip: 'Enter how the theme name will appear in the menu.',
```

Comment:
It's not clear what this means.  "how the theme name will appear" could be about color or location, but I think this is just a name.  So the Name is the long name, and the Menu Label is the short name that appears in the menu?